GitUI/NBugReports/BugReportInvoker.cs

```diff
@@ -100,6 +102,60 @@ public static void Report(Exception exception, bool isTerminating)
                 LogError(exception, isTerminating);
             }
 
+            if (exception is GitDubiousOwnershipSecurityException gitDubiousOwnershipSecurityException)
+            {
+                TaskDialogPage pageSecurity = new()
+                {
+                    Icon = TaskDialogIcon.Error,
+                    Caption = TranslatedStrings.GitSecurityError,
+                    Heading = "Git has detected a security problem that prevent the opening of the repository",
```

Comment:
I'd retain the git's verbage:
```suggestion
                    Heading = "Detected dubious ownership in repository",
```

Reply:
Not 100% convinced by the proposed text because I understand that it could make sense when the user type directly the command but IMHO, it requires a little more context as the command is run by GE.

But I have updated the text to add git's verbage (that I thought I used but in fact it was just in the output logs).

I hope it will be good.